src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -35,34 +36,65 @@ def load_arguments(self, _):
         c.argument('service_runtime_subnet', options_list=['--service-runtime-subnet', '--svc-subnet'], help='The name or ID of an existing subnet in "vnet" into which to deploy the Spring Cloud service runtime. Required when deploying into a Virtual Network.', validator=validate_vnet)
         c.argument('service_runtime_network_resource_group', options_list=['--service-runtime-network-resource-group', '--svc-nrg'], help='The resource group where all network resources for Azure Spring Cloud service runtime will be created in.', validator=validate_node_resource_group)
         c.argument('app_network_resource_group', options_list=['--app-network-resource-group', '--app-nrg'], help='The resource group where all network resources for apps will be created in.', validator=validate_node_resource_group)
-        c.argument('enable_java_agent', is_preview=True, arg_type=get_three_state_flag(), help="Enable java in-process agent", validator=validate_java_agent_parameters)
+        c.argument('enable_java_agent',
+                   arg_type=get_three_state_flag(),
+                   help="Java in process agent is now GA-ed and used by default when Application Insights enabled. "
+                        "This parameter is no longer needed and will be removed in future release.",
+                   validator=validate_java_agent_parameters,
+                   deprecate_info=c.deprecate(target='--enable-java-agent', hide=True))
+        c.argument('app_insights_key',
+                   help="Connection string (recommended) or Instrumentation key of the existing Application Insights.",
+                   validator=validate_tracing_parameters_asc_create)
+        c.argument('app_insights',
+                   help="Name of the existing Application Insights in the same Resource Group. "
+                        "Or Resource ID of the existing Application Insights in a different Resource Group.",
+                   validator=validate_tracing_parameters_asc_create)
+        c.argument('sampling_rate',
+                   type=float,
+                   help="Sampling Rate of application insights. Minimum is 0, maximum is 100.",
+                   validator=validate_tracing_parameters_asc_create)
+        c.argument('disable_app_insights',
+                   arg_type=get_three_state_flag(),
+                   help="Disable Application Insights, "
+                        "if not disabled and no existing Application Insights specified with "
+                        "--app-insights-key or --app-insights, "
+                        "will create a new Application Insights instance in the same resource group.",
+                   validator=validate_tracing_parameters_asc_create)
```

Comment:
Could `validate_tracing_parameters_asc_create` be used as command level validator instead of calling the same validator for multiple parameters?